build.ps1

```diff
@@ -395,6 +395,7 @@ try
         # test categories to exclude
         $testCategory += "&TestCategory!=LongRunning"
         $testCategory += "&TestCategory!=Flaky"
+        $testCategory += "&TestCategory!=InvalidServiceCertificate"
```

Comment:
Add a comment to indicate this is temporary until the pipeline is fixed for docker commands on Linux?

Can we only disable it for Linux?

Reply:
sure